lib/inquisition/rubocop/vulnerability.rb

```diff
@@ -0,0 +1,52 @@
+module Inquisition
+  module Rubocop
+    class Vulnerability
```

Comment:
`Vulnerability` - not quite the right name, would be better to rename it to` Issue`, what do you think?